daal4py/sklearn/_utils.py

```diff
@@ -19,7 +19,7 @@
 import os
 import warnings
 
-from daal4py import _get__daal_link_version__ as dv
+from daal4py._daal4py import _get__daal_link_version__ as dv
```

Comment:
I think this changes are not relevant. `_get__daal_link_version__` attribute is in `daal4py`. Could you please share issue details?

Reply:
When I tried to import it from `src/gbt_model_builder.pyx`, I got a cyclic import error. This fix solved the problem.